tests/unit/test_exec.py

```diff
@@ -1,9 +1,75 @@
 __copyright__ = "Copyright (c) 2020-2021 Jina AI Limited. All rights reserved."
 __license__ = "Apache-2.0"
 
+import numpy as np
+
+from jina import Document, DocumentArray
 from jina.executors import BaseExecutor
+#from jinahub.segmenter.vggish_audio_segmenter import VGGishSegmenter
+
+import sys
+sys.path.insert(1, '../..')
```

Comment:
Please use sys.path with current folder info (and avoid `path_a/path_b`. To do this use `path.join(path_a, path_b)`

Reply:
This code won't go into master